src/everest/bin/utils.py

```diff
@@ -214,6 +215,17 @@ def update(self, status: dict[str, Any]) -> None:
                     self._clear_lines = 0
             if SIM_PROGRESS_ID in status:
                 match status[SIM_PROGRESS_ID]:
+                    case EndEvent(msg=msg) as end_event:
+                        if end_event.failed:
+                            print(
+                                f"Everest run {Fore.RED}failed{Fore.RESET} with: {msg}"
```

Comment:
Is this replacing `print(end_event.msg)` below in client.py? Or maybe something else in `utils.py` where there are colours in use.